lightning/src/ln/shutdown_tests.rs

```diff
@@ -400,6 +401,11 @@ fn updates_shutdown_wait() {
 
 #[test]
 fn htlc_fail_async_shutdown() {
+	do_htlc_fail_async_shutdown(true);
+	do_htlc_fail_async_shutdown(false);
+}
+
+fn do_htlc_fail_async_shutdown(blinded_recipient: bool) {
```

Comment:
Why test this change in `shutdown_tests`?

Reply:
Just to keep the tests DRY, it's a somewhat long test.